### PR TITLE
dnd-character: add exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -940,6 +940,21 @@
...
         ],
         "difficulty": 4
+      },
+      {
+        "slug": "dnd-character",
+        "name": "D&D Character",
+        "uuid": "65d80b55-02f0-4d40-a530-79843553252f",
+        "practices": [
+          "int",
+          "random",
```
Comment (lpil) [inline config.json:950]: Correct me if I'm wrong, but shouldn't all the practices be in the prerequisites also? Have to have learnt it to practice it?

**Review by lpil — APPROVED** [review]
Looks excellent. A fun exercise too :)

I've left 1 comment inline

File: config.json (hunk 1 above)
Comment (ErikSchierboom) [reply config.json:950]: Yep, my mistake.

File: config.json [hunk 2]
```diff
@@ -940,6 +940,21 @@
...
+          "custom-type"
+        ],
+        "prerequisites": [
+          "int",
+          "custom-type"
```
Comment (ErikSchierboom) [inline config.json:955]:
```suggestion
          "random",
          "custom-type"
```

**Pull request merged**